server/docker-compose.yml

```diff
@@ -12,6 +12,8 @@ api:
     - INITIAL_ADMIN_CODE=initialadmincode
   links:
    - mongodb
+  labels:
+   - "io.kontena.container.skip_logs=1"
```

Comment:
Yup, but having a production server melt down if you enable DEBUG logging is worse.